syntax = "proto3";
package company_service.proto;

message CompanyUser {
  uint64 id = 1;
  uint64 company_id = 2;
  uint64 user_id = 3;
  string state = 4;
  string type = 5;
  optional string about = 6;
  bool contract = 7;
  bool is_active = 8;
}